shanoir-ng-datasets/src/main/java/org/shanoir/ng/dicom/web/StudyInstanceUIDHandler.java

```diff
@@ -19,6 +19,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
```

Comment:
Hi @DuckflipXYZ, last minor thing: please remove unnecessary import and we are good. Cheers, Michael